### PR TITLE
Addition of public meetings page

### PR DESCRIPTION
**Pull request by james-malone**
Added new page for public meetings:
- Past meetings with links to content & notes
- Future meetings

### CODE REVIEW

**Comment by evilsoapbox** [comment]
Added update for second meeting - sorry for the second commit.

File: _pages/public-meetings.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  {% endfor %}
+</table>
+*This list was last updated on {{ site.data.meetings.last_updated }}.*
+
+All Apache Beam meetings are open to the public and we encourage anyone to attend. From time to time space in our event location may be limited, so preference will be given to PPMC members and others on a first-come, first-serve basis.
+
+## I want to give a public talk about Apache Beam
+To get started, we recommend you review the Apache Beam [presentation materials]({{ site.baseurl }}/presentation-materials/) page to review the content the Apache Beam community has already created. These materials will possibly save you time and energy as you create content for your event.
+
+Once you have scheduled your event, we welcome you to announce it on the [dev@beam.incubator.apache.org](mailto:dev@beam.incubator.apache.org) mailing list. Additionally, please open a [JIRA item](https://issues.apache.org/jira/browse/BEAM) using the `website` component with details so we can update this page.
```
Comment (takidau) [inline _pages/public-meetings.md]: Wouldn't public talks be better announced on the user list?

Comment (evilsoapbox) [inline _pages/public-meetings.md]: Good point. Fixing (new commit incoming)

**Comment by takidau** [comment]
LGTM, thanks!

**Pull request merged**